fix: revise words on docs intro page (#9588)
medusajs/medusa: www/apps/book/app/page.mdx

# {pageNumber} Introduction

Medusa is a digital commerce platform with a built-in framework for customization.

> Medusa is ships with three main tools:

Reviewer: ```suggestion
Medusa ships with three main tools:
```

1. A suite of commerce modules that provide core commerce functionalities, such as tracking inventory, calculating cart totals, accepting payments, managing orders, and much more.

> 2. A framework to build custom functionality that is specific to your business, product, or industry. This includes tools for introducing custom API endpoints, business logic, and data models; building workflows and automations; and integrating with third-party services.

Reviewer: ```suggestion
2. A framework to build custom functionalities specific to your business, product, or industry. This includes tools for introducing custom API endpoints, business logic, and data models; building workflows and automations; and integrating with third-party services.
```

3. A customizable admin dashboard for merchants to configure and operate their store.

> When you start Medusa the first time you get a fully fledged commerce platform with all the features you need to get off the ground. But unlike other platforms, Medusa is built with customization in mind. This avoids the need to build hacky workarounds that are difficult to maintain and scale, and instead allows you to focus on building features that move the needle.

Reviewer: ```suggestion
When you start Medusa the first time you get a fully fledged commerce platform with all the features you need to get off the ground.

But, unlike other platforms, Medusa is built with customization in mind. You don't need to build hacky workarounds that are difficult to maintain and scale. Instead, you can focus on building features that move the needle.
```

Not a fan of " you can focus on building features that move the needle." as it feels vague. What do you think of something like " you can focus on building features that brings your business's vision to life"?

---

## Who should use Medusa

> Medusa is for businesses that have unique requirements that other digital commerce platforms can't easily support. Businesses of any size can use Medusa, from small startups to large enterprises - all it takes is a developer who can run a few commands and deploy your project.

Reviewer: ```suggestion
Medusa is for businesses with unique requirements that other digital commerce platforms can't easily support. Businesses of any size can use Medusa, from small startups to large enterprises - all it takes is a developer to manage and deploy your project.
```

"a developer who can run a few commands" feels somewhat demeaning 😅

Below are some stories from companies that use Medusa:

<CardList items={[

>   {

Author: would really like these to use logos of the companies to make them feel more user story-ish.

Reviewer: I can handle this in a separate PR 👍🏻
